src/CassandraUtil.js

```diff
@@ -64,7 +65,7 @@ module.exports = class CassandraUtil extends events.EventEmitter {
         let largestOffset = null
         this.client.stream(query, queryParams, {
             prepare: true, autoPage: true,
-        }).on('readable', () => {
+        }).on('readable', function () {
```

Comment:
Looks like a bugfix. I wonder how this has worked previously? Could we add tests that expose the bug that previously existed (i.e. a test that fails without this change)?

(This is how I read this change): this needs to be `function` (instead of an arrow function), because it reads into `this` which in this case is expected to be bound to something else than what the arrow function would bind it to (namely, `CassandraUtil`).

Reply:
BTW eslint probably won't like using `function`s, but you can add a line-level rule exception to suppress the error.

Reply:
Yes I saw this bug on `refactor-oom-protocol`. The opposite change was made (from `function ()` to `() =>`) and this made `CassandraUtil` fail.

Yes exactly, the arrow was redefining the context and `this` didn't refer to the same object anymore.

Line-level rule exception added.